zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/FastLeaderElection.java

```diff
@@ -838,11 +839,11 @@ synchronized public Vote getVote(){
      */
     private ServerState learningState(){
         if(self.getLearnerType() == LearnerType.PARTICIPANT){
-            LOG.debug("I'm a participant: " + self.getId());
+            LOG.debug("I am a participant: {}", self.getId());
```

Comment:
Sorry, I'm not a native but why do you need this language change?

Reply:
This is called a 'contraction'.  A contraction is a shortened version of the written and spoken forms of a word, syllable, or word group, created by omission of internal letters and sounds.

There is no hard-and-fast rule, but I generally encourage folks to avoid them, especially when using them in application output of any kind.

`Only use them in "conversational" parts of the documentation (i.e. explanatory text).`

https://docs.ubuntu.com/styleguide/en#contractions
https://developers.google.com/style/contractions